data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -430,7 +454,7 @@ spec:
                 - name: CPU_PLUGIN_IMAGE
                 - name: OPERATOR_VERSION
                   value: 0.14.0
-                image: quay.io/kubevirt/ssp-operator:latest
```

Comment:
Needs to be reverted

Reply:
Gah and this is why we shouldn't generate in the tree, thanks!